packages/cxl-ui/scss/cxl-card.scss

```
@@ -32,6 +36,157 @@
   background-color: var(--lumo-contrast);
 }
 
+/**
+ * CXL 2.0
+*/
+
+:host([theme="cxl-dashboard"]) {
```

There might be multiple themes applied at the same time

```suggestion
:host([theme~="cxl-dashboard"]) {
```